docs/test_framework_integration/junit_5.md

```diff
@@ -8,6 +9,43 @@ The extension supports two modes:
 - containers that are restarted for every test method
 - containers that are shared between all methods of a test class
 
+
+## Extension
+
+Jupiter integration is provided by means of the `@Testcontainers` annotation.
```

Comment:
The annotation lives in a separate module (unlike JUnit 4 integration), worth adding module coordinates here

Reply:
👍